### DISCUSSION TITLE
Cannot create full width Toggle Button group

### DISCUSSION THREAD

**Discussion by danswiser** [filamentphp/filament#11168] [Help]
### Package

Form builder

### Package Version

v3.2.20

### How can we help you?

The `->class(['w-max'])` line of the grouped ToggleButtons view prevents me from creating a full width component.

I intended on adding `->extraAttributes(['class' => 'w-full'])` when creating the component but it seems like the 'w-max' class takes precedence.

https://github.com/filamentphp/filament/blob/d88e96f3b429840dd2417a1730365f5dedb3a7af/packages/forms/resources/views/components/toggle-buttons/grouped.blade.php#L23-L29

Is there a preferred way to make ToggleButtons full-width?

**Comment by alguintu** [comment]
I encountered this recently and used a custom css that removes the w-max from the group.

```
.negate-w-max {
    width: 100% !important;
}
```

Publish your css as usual and add it as `extraAttributes` to your ToggleButton.

```
// in boot()

FilamentAsset::register([
            Css::make('my-styles', __DIR__ . '/../../resources/css/my-styles.css'),
        ]);
```

```
ToggleButtons::make('my_toggle')
     ->grouped()
     ->extraAttributes(['class' => 'negate-w-max'], true)
```

**Comment by howdu** [comment]
Another option is to overwrite with style

```php
ToggleButtons::make('foo')
        ->extraAttributes(['style' => 'width: 100%'])
```
